### ISSUE TITLE
Tenant dropdown needed on ARM API test drive

### ISSUE THREAD

**Issue by joshbooker** [Azure/azure.github.io#31]
ARM API test drive may show no subscriptions for credentials having multiple tenancies. If wrongly tenancies is picked for auth then nothing works. A drop down for tenancies and auth getting auth token from chosen tenancy should work.

**Comment by devigned** [comment]
You are exactly right. Thank you for logging the issue. I think I'm going to have rewrite some of the security logic in SwaggerUI to deal with OAuth and tenants in AAD. I'll probably get to this in the next week or so.

If you have more than one tenant, the token received back from Azure Active Directory is for the common tenant and will not be Tenant specific (thus no subscriptions returned). If you use an organizational id or a live id with only one tenant, then you will get back the tenant specific token (thus listing subscriptions).

Basically, to solve this, I would need to request the token from the common tenant, then request the list of subscriptions per tenant, then override SwaggerUI to use the proper tenant per subscription. 

:disappointed:

**Comment by joshbooker** [comment]
i do get all tenants returned. Wonder if it would work to populate tenant drop down from common token. Then re auth with chosen tenant to get list of subs. Not sure if that works or simplifies scope of changes.
